src/main/groovy/org/pih/warehouse/product/ProductSupplierListParams.groovy

```diff
@@ -0,0 +1,24 @@
+package org.pih.warehouse.product
+
+import org.pih.warehouse.api.PaginationParams
+
+class ProductSupplierListParams extends PaginationParams {
```

Comment:
@jmiranda to think about naming here.

Reply:
Ok just change the name to PaginationCommand and ProductSupplierFilterCommand. 

I think pagination should support totalCount as well. 

In fact, for the pagination command, use this implementation as a guide.

https://github.com/bmihovski/grails3asciibooks/blob/64e1b6ee69260ae15af0a19ad409b5733ef9b756/asciibooks/src/main/groovy/com/asciibooks/commands/PaginationCommand.groovy

We probably don't need to handle sort/order now (that probably needs a separate class like SearchCommand anyway).

Reply:
@jmiranda thanks for the review. I will checkout the example you've attached. As for the `totalCount`, it would be difficult to include it, because it's not a user that we receive this param from, but it's counted after running the `createCriteria`, so I kinda do not see a use case for that currently.